compiler_opt/es/blackbox_optimizers.py

```diff
@@ -595,7 +606,10 @@ class TrustRegionSubproblemOptimizer(object):
    [1] Except for one `bad' case which does not occur almost surely.
   """
 
-  def __init__(self, model_function, trust_region_params, x_init=None):
+  def __init__(self,
+               model_function: QuadraticModel,
+               trust_region_params: Dict[str, Any],
```

Comment:
Mapping[str, Any]

Reply:
This instance requires the ability to update the dictionary so it has been left as Dict[str, Any], but the other instance has been changed to Mapping[str, Any]